src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE books (
+    id BIGINT(11) NOT NULL AUTO_INCREMENT,
+    title VARCHAR(45) NOT NULL,
```

Comment:
Why are you restricting your columns so much? I bet there is longer book name than 45 symbols for sure. The same with BIGINT(11).